path_config.go

```diff
@@ -245,13 +245,11 @@ func (b *jwtAuthBackend) pathConfigWrite(ctx context.Context, req *logical.Reque
 		return logical.ErrorResponse("'oidc_discovery_url' must be set for OIDC"), nil
 
 	case config.JWKSURL != "":
-		ctx, err := b.createCAContext(context.Background(), config.JWKSCAPEM)
+		keyset, err := jwt.NewJSONWebKeySet(ctx, config.JWKSURL, config.JWKSCAPEM)
```

Comment:
It looks like the `keyset.VerifySignature(ctx, testJWT)` will no longer use a context with the config's CA.

I wonder if the `VerifySignature` call in here should be storing and using the `caCtx` that got instantiated in `NewJSONWebKeySet`.

Reply:
The `keyset.VerifySignature()` call below actually does end up using the context with the config's CA that's created in cap/jwt (see [keyset.go#L114](https://github.com/hashicorp/cap/blob/main/jwt/keyset.go#L114)). That's because the underlying `remoteKeySet` stores the context (see [jwks.go#L49](https://github.com/coreos/go-oidc/blob/v2/jwks.go#L49)) and uses the associated client with the config's CA to make the HTTP request for the JWKS (see [jwks.go#L198](https://github.com/coreos/go-oidc/blob/v2/jwks.go#L198)). The context passed to `keyset.VerifySignature()` isn't used for the JWKS request.

Reply:
Ah yep, you're right! Thanks for the pointer on `updateKeys()`.